[UI] ui.test_role.RoleTestCase.test_positive_assign_cloned_role - TypeError

There seems to be a bug in our test code. The test was skipped due to BZ up until now.

```
    @skip_if_bug_open('bugzilla', 1353788)
    @tier1
    def test_positive_assign_cloned_role(self):
        """Clone role and assign it to user
    
            @id: cbb17f37-9039-4875-981b-1f427b095ed1
    
            @expectedresults: User is created successfully
    
            @BZ: 1353788
            """
        user_name = gen_string('alpha')
        role_name = gen_string('alpha')
        with Session(self.browser) as session:
            # Clone one of the builtin roles
            self.role.clone(choice(ROLES), role_name)
            # Create user wit this role
            make_user(
                session, username=user_name, roles=[role_name], edit=True)
            self.user.search_and_click(user_name)
            self.user.click(tab_locators['users.tab_roles'])
            element = self.user.wait_until_element(
>               common_locators['entity_deselect'] % role_name)
E           TypeError: unsupported operand type(s) for %: 'tuple' and 'unicode'
```

--- a/tests/foreman/ui/test_role.py
+++ b/tests/foreman/ui/test_role.py
@@ -215,8 +215,9 @@ class RoleTestCase(UITestCase):
                 session, username=user_name, roles=[role_name], edit=True)
             self.user.search_and_click(user_name)
             self.user.click(tab_locators['users.tab_roles'])
+            strategy, value = common_locators['entity_deselect']
             element = self.user.wait_until_element(
-                common_locators['entity_deselect'] % role_name)
+                (strategy, value % role_name))
             self.assertIsNotNone(element)
 
     @skip_if_bug_open('bugzilla', 1353788)

